### DISCUSSION TITLE
counting with moving camera

### DISCUSSION THREAD

**Discussion by gbo999** [tryolabs/norfair#257] [General]
when tracking with camera motion, if i want to count objects in a scene, object will count only once with this feature, even if they were in one frame and not in another and then appears again?

**Comment by facundo-lezama** [comment]
Hi @gbo999! I'm not sure I understand the question. Can you please give a more detailed explanation?

**Reply by gbo999** [reply]
o.k I am attaching a video of me passing through a number of cups. I want to count the total number of cups in the video, even though you could see some of the more than once on the way back. how can i achieve this mechinsm?

https://user-images.githubusercontent.com/31617142/222547043-13540016-8987-4c45-9e06-65e7f5845606.mp4

**Reply by facundo-lezama** [reply]
As described in [this demo](https://github.com/tryolabs/norfair/tree/master/demos/camera_motion), when using _Homographies_, you could track objects even when the camera is moving and hence, count them. But please consider that in complex scenarios, as the one of the video you sent (mostly when turning around), it is not easy to have a good performance.
